src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -1084,7 +1084,7 @@ ScaledDotProductAttention::ScaledDotProductAttention(const std::shared_ptr<ov::N
     : Node(op, context, SDPAShapeInferFactory(op)) {
     std::string errorMessage;
     if (!isSupportedOperation(op, errorMessage)) {
-        OPENVINO_THROW("CPU: " + errorMessage);
+        THROW_CPU_NODE_ERR(errorMessage);
```

Comment:
May be throw not implemeted? Could you please unify this error message across other nodes?

Reply:
Done